src/modules/app/createNavigateToEvent.js

```diff
@@ -22,7 +22,7 @@ export default (dispatch: Dispatch<StoreActionType>, navigation: NavigationScree
     key
   })
 
-  return dispatch({
+  dispatch({
```

Comment:
I think we want to return the dispatch here. Was there a error with a return type?

Reply:
Maybe I'm wrong: https://github.com/Integreat/integreat-react-native-app/pull/81#discussion_r296856858